api/envoy/extensions/filters/http/compressor/v3/compressor.proto

```diff
@@ -21,40 +21,94 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Compressor :ref:`configuration overview <config_http_filters_compressor>`.
 // [#extension: envoy.filters.http.compressor]
 
-// [#next-free-field: 7]
+// [#next-free-field: 9]
 message Compressor {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.http.compressor.v2.Compressor";
 
+  message CommonDirectionConfig {
+    // Runtime flag that controls whether the filter is enabled for compression or not. If set to false, the
+    // filter will operate as a pass-through filter. If the field is omitted, the filter will be enabled.
+    config.core.v3.RuntimeFeatureFlag enabled = 1;
+
+    // Minimum response length, in bytes, which will trigger compression. The default value is 30.
+    google.protobuf.UInt32Value content_length = 2;
```

Comment:
`min_content_length`?

Reply:
Yeap. This is a good opportunity to make the name clearer. Renamed.